packages/chai-openapi-response-validator/lib/openapi-validator/lib/classes/RequestPromiseResponse.js

```diff
@@ -4,7 +4,7 @@ class RequestPromiseResponse extends AbstractResponse {
   constructor(res) {
     super(res);
     this.status = res.statusCode;
-    this.body = res.body.replace(/"/g, '');
+    this.body = res.request.headers.accept === 'application/json' ? res.body : res.body.replace(/"/g, '');
```

Comment:
```suggestion
    this.body = res.request._json // eslint-disable-line no-underscore-dangle
      ? res.body
      : res.body.replace(/"/g, '');
```

Reply:
I noticed using `_json` would also work, but you choose which is clearer

Reply:
I like `._json` better, thanks for suggestion.